raiden/raiden_service.py

```diff
@@ -1024,14 +1024,14 @@ def connection_manager_for_token_network(
         self, token_network_address: TokenNetworkAddress
     ) -> ConnectionManager:
         if not is_binary_address(token_network_address):
-            raise InvalidAddress("token address is not valid.")
+            raise InvalidBinaryAddress("token address is not valid.")
 
         known_token_networks = views.get_token_network_addresses(
             views.state_from_raiden(self), self.default_registry.address
         )
 
         if token_network_address not in known_token_networks:
-            raise InvalidAddress("token is not registered.")
+            raise InvalidBinaryAddress("token is not registered.")
```

Comment:
That seems not to fit the definition of that exception?! Maybe here it should be the `ValueError`

`Raised when the address is not binary or it is not 20 bytes long.`